### PR TITLE
Rebecca SteelHall.MAUI

### PR DESCRIPTION
**Pull request by Rebecca-Coding21**
# Description

I implemented an example program for C# WebService using .NET MAUI. The example program creates a steel hall in RFEM. The user define values for frame height, frame span, frame distance, frame number and rood angle. Otherwise default values are used for calculation. The buttons give the ability to start the calculation, export the results as csv-file, close the current model or close the application.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [ ] Unit Tests
- [ ] Attached examples
- [x] Own example attached to Pull Request

* RFEM version: 6.02.0070
* .Net SDK: .NET 7.0

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jarabroz — COMMENTED** [review]
We can define target platform just for Windows.

File: examples/SteelHall.MAUI/SteelHall - Backup.MAUI.csproj [hunk 1]
```diff
@@ -0,0 +1,76 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (jarabroz) [inline examples/SteelHall.MAUI/SteelHall - Backup.MAUI.csproj:1]: File not needed

File: examples/SteelHall.MAUI/App.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        //const int width = (int)MainPage.WidthRequest;
+        //const int height = (int)MainPage.HeightRequest;
+
+        public App()
+        {
+            InitializeComponent();
+
+
+            MainPage = new AppShell();
+            //            var width = (int)MainPage.WidthRequest;
```
Comment (jarabroz) [inline examples/SteelHall.MAUI/App.xaml.cs:23]: If not necessary remove commented code

**Review by jarabroz — CHANGES_REQUESTED** [review]
Define only build for Win platform as active

**Review by jarabroz — CHANGES_REQUESTED** [review]
Finished review. Just make na WIN app

File: examples/SteelHall.MAUI/SteelHall - Backup.MAUI.csproj (hunk 1 above)
Comment (Rebecca-Coding21) [reply examples/SteelHall.MAUI/SteelHall - Backup.MAUI.csproj]: Done.

File: examples/SteelHall.MAUI/App.xaml.cs (hunk 2 above)
Comment (Rebecca-Coding21) [reply examples/SteelHall.MAUI/App.xaml.cs]: Done.

**Review by jarabroz — APPROVED** [review]

**Pull request merged**